docs/data-sources/data-source--enum.md

```diff
@@ -22,3 +22,41 @@ Select the desired enumeration, by selecting the containing .NET assembly, and t
 The value returned from the List editor is the configured enumeration type.
 
 Depending on the `List editor` used, this may be wrapped in a `List<T>`.
+
+##### Controlling Enum display in Contentment
```

Comment:
I'd remove the "in Contentment" bit, as the documentation is already in context of the package.